packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -38,7 +38,7 @@ for PACKAGE in ${PACKAGES[@]}; do
         # It's a server, start the pm2 process
         echo "Starting ${PACKAGE}"
         yarn build
-        pm2 start --name $PACKAGE dist --wait-ready --listen-timeout 15000 --time
+        pm2 start --name $PACKAGE 'yarn start' --wait-ready --listen-timeout 15000 --time
```

Comment:
May need to do some testing on this to ensure it performs just as well as it previously has (ie. automatic and manual restart work fine, memory usage consistent, etc.)

Made the change because entity-server no longer has its main file be simply `index.js` under the dist directory, so the previous start script wasn't launching the server. Generally I prefer this method as it allows us to be a bit more custom about how various servers are launched (ie. we can specify whatever args/pre-start steps we want in package.json) but gotta make sure that pm2 plays nicely with that 😄